compiler/fir/resolve/src/org/jetbrains/kotlin/fir/resolve/transformers/body/resolve/FirExpressionsResolveTransformer.kt

```diff
@@ -69,6 +68,7 @@ open class FirExpressionsResolveTransformer(transformer: FirBodyResolveTransform
         qualifiedAccessExpression.transformAnnotations(this, data)
         qualifiedAccessExpression.transformTypeArguments(transformer, ResolutionMode.ContextIndependent)
 
+        val wasField = (qualifiedAccessExpression.calleeReference as? FirNamedReference)?.name == Name.identifier("field")
```

Comment:
There is the constant `NAME_FOR_BACKING_FIELD` in `compiler/fir/tree/.../DefaultNames.kt`.

Reply:
Good to know that, thanks.